_posts/0000-01-02-Lilynette.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+
+Je suis enthousiaste face à notre avenir et ce que nous pouvons accomplir ensemble, cette année et dans le futur.
+
```

Comment:
Nevermind

I'd love to play this for my girlfriend.
Unfortunately i can't play the piano and i have no girlfriend..

Loool